src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentationOptions.cs

```diff
@@ -97,6 +97,9 @@ public class SqlClientInstrumentationOptions
         /// <c>SqlCommand</c>s regardless of <see cref="CommandType"/>. Use
         /// SetDbStatementForText to capture Text, StoredProcedure, and all
         /// other command text.</item>
+        /// <item>When using <c>System.Data.SqlClient</c> on .NET Framework, the
+        /// instrumentation will only capture <c>sqlCommand.CommandText</c> for
+        /// <see cref="CommandType.StoredProcedure"/> commands.</item>
```

Comment:
@vishweshbankwar I think this would be better as a sub-item to the .NET Framework bullet. Here's my stab at it:

```xml
        /// <summary>
        /// Gets or sets a value indicating whether or not the <see
        /// cref="SqlClientInstrumentation"/> should add the text of commands as
        /// the <see cref="SemanticConventions.AttributeDbStatement"/> tag.
        /// Default value: <see langword="false"/>.
        /// </summary>
        /// <remarks>
        /// <para>
        /// <b>WARNING: SetDbStatementForText will capture the raw
        /// <c>CommandText</c>. Make sure your <c>CommandText</c> property never
        /// contains any sensitive data.</b>
        /// </para>
        /// <para><b>SetDbStatementForText is supported on all runtimes.</b>
        /// <list type="bullet">
        /// <item>On .NET and .NET Core SetDbStatementForText only applies to
        /// <c>SqlCommand</c>s with <see cref="CommandType.Text"/>.</item>
        /// <item>On .NET Framework SetDbStatementForText applies to all
        /// <c>SqlCommand</c>s regardless of <see cref="CommandType"/>.
        /// <list type="bullet">
        /// <item>When using <c>System.Data.SqlClient</c> use
        /// SetDbStatementForText to capture StoredProcedure command
        /// names.</item>
        /// <item>When using <c>Microsoft.Data.SqlClient</c> use
        /// SetDbStatementForText to capture Text, StoredProcedure, and all
        /// other command text.</item>
        /// </list></item>
        /// </list>
        /// </para>
        /// </remarks>
```

Reply:
updated